zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/message/MessageSubscriptionCorrelateProcessor.java

```diff
@@ -86,45 +81,30 @@ public void processRecord(final TypedRecord<MessageSubscriptionRecord> record) {
       // This concerns the acknowledgement of a retried correlate process message subscription
       // command. The message subscription was already marked as correlated for this message, and
       // another message has started correlating. There's no need to update the state.
-      LOG.warn(
-          """
-          Expected to acknowledge correlating message with key '{}' to subscription with key '{}' \
-          but the subscription is already correlating to another message with key '{}'""",
-          record.getValue().getMessageKey(),
-          subscription.getKey(),
-          subscription.getRecord().getMessageKey());
       final var reason =
-          String.format(
-              NO_SUBSCRIPTION_FOUND_MESSAGE,
-              record.getValue().getElementInstanceKey(),
-              BufferUtil.bufferAsString(record.getValue().getMessageNameBuffer()));
-      rejectionWriter.appendRejection(record, RejectionType.NOT_FOUND, reason);
+          """
+          Expected to acknowledge correlating message with key '%d' to subscription with key '%d' \
+          but the subscription is already correlating to another message with key '%d'"""
```

Comment:
🤡 You could consider pulling this out into a constant like we did with the other error message